Lib/test/test_capi.py

```diff
@@ -1017,6 +1017,14 @@ def test_state_access(self):
                 with self.assertRaises(TypeError):
                     increment_count(1, 2, 3)
 
+    def test_Py_CompileString(self):
```

Comment:
Actually, while backporting this patch to our internal 3.9, I noticed this test method is in the wrong class. It should be in the CAPITest class.

Reply:
I will submit a fix today